src/components/forms/TextField.js

```diff
@@ -60,11 +55,17 @@ export default {
       const inputLength = (this.inputValue && this.inputValue.toString() || '').length
       let min = inputLength
 
-      if (this.min !== 0 && inputLength < this.min) {
-        min = this.min
+      if (this.counterMin !== 0 && inputLength < this.counterMin) {
```

Comment:
@johnleider Good catch--looks like this is a bug in the current version as well. Take a look at the revised computed properties for `counterMin` and `counterMax`. Or let me know if you have another way you prefer to handle this issue.